src/styles/_custom.css

```diff
@@ -1,3 +1,11 @@
+html {
+  height: 100%;
+}
 body {
-  background-color: var(--blue);
+  background-color: var(--light-gray);
+  height: 100%;
+}
+
+#root {
+  height: 100%;
```

Comment:
How come you set the height to 100%?

Reply:
oops, was trying something and forgot it there, that explains some of the weird behavior